Beis.LearningPlatform.Web/Views/Shared/Components/CmsNextPageRadioButton/Default.cshtml

```diff
@@ -0,0 +1,98 @@
+                    <label for="cms_component_NextPageRadioButton_rdo1" class="govuk-label govuk-radios__label"><strong></strong>@Model.Radio1Text</label>
+                </div>
+
+                <div class="govuk-radios__item">
+                    <input id="cms_component_NextPageRadioButton_rdo2" type="radio" value="2" class="govuk-radios__input" name="cms_component_NextPageRadioButton_rdon">
+                    <label for="cms_component_NextPageRadioButton_rdo2" class="govuk-label govuk-radios__label"><strong></strong>@Model.Radio2Text</label>
+                </div>
+
+                <div class="govuk-radios__item">
+                    <input id="cms_component_NextPageRadioButton_rdo3" type="radio" value="3" class="govuk-radios__input" name="cms_component_NextPageRadioButton_rdon">
+                    <label for="cms_component_NextPageRadioButton_rdo3" class="govuk-label govuk-radios__label"><strong></strong>@Model.Radio3Text</label>
+                </div>
+
+            </div>
+
+            <div class="beis-text govuk-!-padding-top-6" id="anchor3">
+                <div class="">
+                    <div class="govuk-width-container">
+                        <div class="govuk-grid-row">
+
+                            <p class="govuk-button-group">
+                                <a id="cms_component_NextPageRadioButton_btn1" href="@Model.Radio1Url" style="display:" role="button" draggable="false" class="govuk-button govuk-button--start primary " data-module="govuk-button">
+                                    @Model.ButtonText
+                                </a>
+                                <a id="cms_component_NextPageRadioButton_btn2" href="@Model.Radio2Url" style="display:none;" role="button" draggable="false" class="govuk-button govuk-button--start primary " data-module="govuk-button">
+                                    @Model.ButtonText
+                                </a>
+                                <a id="cms_component_NextPageRadioButton_btn3" href="@Model.Radio3Url" style="display:none" role="button" draggable="false" class="govuk-button govuk-button--start primary " data-module="govuk-button">
+                                    @Model.ButtonText
+                                </a>
+                            </p>
+
+
+
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+
+
+        </div>
+        <noscript>
+            <div class="beis-white govuk-grid-column-two-thirds">
+                <div class="  govuk-!-padding-bottom-6">
+
+
+                    <p class="govuk-button-group">
+                    </p>
+
+
+                    <div class="  ">
+                        <div>
+                            <span class="beis-link-icon">
+                                <img src="/assets/images/link-arrow.svg" class="svgimg beis-link-icon " width="23" height="24">
```

Comment:
please make repested items into components e.g. this link arrow is repeated frequently

Reply:
DONE in last commit
Created new component